### ISSUE TITLE
ncnn

### ISSUE THREAD

**Issue by Leaf-G** [MichiganLu/Superpoint_android#2]
I find this file--'coco_pretrained.param' include softmax, PixelShuffle and so on. 
Could you tell me the reason?
wheather  are you eidt the net of superpoint before you train this net?
or other reason.
Looking
 forward your reply.

**Comment by MichiganLu** [comment]
I believe this is the original superpoint network from the pytorch implementation repo, not the tensorflow one, I did not do any editing for the NCNN version whatsoever.

**Comment by Leaf-G** [comment]
thanks,
I successfully solved the problem.
But when I try to generate superpoint.onnx by superpoint_bn.py(https://github.com/shaofengzeng/SuperPoint-Pytorch/blob/master/model/superpoint_bn.py), these are some errors happend.
The log is here:
RuntimeError("ONNX export failed on an operator with unrecognized namespace {}::{}. ")
Dou you know the reason? Or how do you generate this .onnx file successfully?
Looking forward your reply.
Thank you for your help!
